### DISCUSSION TITLE
Different color audio / motion

### DISCUSSION THREAD

**Discussion by mimainka** [blakeblackshear/frigate#13059] [Ideas]
Hi,

on one of my cameras I also have audio detection enabled next to motion as I want to know when my (not-smart) doorbell rang during times i'm not at home.
With the new 0.14 UI you have given us a very beautiful and clear view on motion in the review area.
It makes it really easy to scroll through the day and see when something happened by the size of the yellow line and if Frigate detected it or if it was something completely different.

What is a bit of a work is to divide between real motion in terms of pixel change and audio that caused the bar to appear bigger in the timeline. From what I experienced, audio seems to create bars as well and depending on the settings of the camera (noise reduction, microphone volume) and inside the frigate config (min_volume), those bars might be there a lot.

I get that audio is considered motion according to the docs.

But would it be possible in a future release to give audio a different color (red or something), so it can be easily said: Hey that's not motion, thats audio, so I can check what exactly it was and maybe i have to include it in my detect-list?

**Comment by NickM-27** [comment]
This is incorrect, audio does not create bars. In the early days of this UI audio was included as blue but it became clear that audio information was not helpful because it was not possible to listen to it so it was removed entirely
